src/xtopcom/xrpc/xrpc_init.cpp

```diff
@@ -36,8 +36,8 @@ xrpc_init::xrpc_init(std::shared_ptr<xvnetwork_driver_face_t> vhost,
     case common::xnode_type_t::consensus_auditor:
         assert(nullptr != txpool_service);
         init_rpc_cb_thread();
-        m_cluster_handler = std::make_shared<xcluster_rpc_handler>(vhost, router_ptr, txpool_service, store, block_store, txstore,  make_observer(m_thread));
-        m_cluster_handler->start();
+        m_rpc_handler = std::make_shared<xrpc_handler>(vhost, router_ptr, txpool_service, store, block_store, txstore,  make_observer(m_thread));
+        m_rpc_handler->start();
```

Comment:
if memory serves me, these code could be removed?
@wishwen knows more about this.